### ISSUE TITLE
Failed to get sectionGroup

### ISSUE THREAD

**Issue by mazharcoder** [streetwriters/notesnook-importer#42]
`1. Failed to get sectionGroup (original error: The specified resource ID does not exist.). (Request failed with status code: 404). 
2. Failed to get sectionGroup (original error: The specified resource ID does not exist.). (Request failed with status code: 404).
3. Failed to get sectionGroup (original error: The specified resource ID does not exist.). (Request failed with status code: 404).
4. Failed to get sectionGroup (original error: The specified resource ID does not exist.). (Request failed with status code: 404).
5. Failed to get sectionGroup (original error: The specified resource ID does not exist.). (Request failed with status code: 404).
6. Failed to get sectionGroup (original error: The specified resource ID does not exist.). (Request failed with status code: 404).
7. Failed to get sectionGroup (original error: The specified resource ID does not exist.). (Request failed with status code: 404).
8. Failed to get sectionGroup (original error: The specified resource ID does not exist.). (Request failed with status code: 404).
9. Failed to get sectionGroup (original error: The specified resource ID does not exist.). (Request failed with status code: 404).
10. Failed to get sectionGroup (original error: The specified resource ID does not exist.). (Request failed with status code: 404). `

**Comment by mazharwhite** [comment]
This happened when importing OneNote Data

**Comment by thecodrr** [comment]
Hey, this can safely be ignored. Just check if you have all the notes imported.

**Comment by mazharwhite** [comment]
Hi @thecodrr 

I did check and nothing was imported from OneNote.

**Comment by thecodrr** [comment]
Nothing at all? That's weird.

**Comment by mazharwhite** [comment]
Yes, I checked first and retried the process but the same issue occurred.

**Comment by thecodrr** [comment]
Does the progress show anything?

**Comment by mazharwhite** [comment]
I just get the above error. I did this yesterday. Let me try again

**Comment by mazharwhite** [comment]
@thecodrr Any update on this?
